avalon/nuke/workio.py

```diff
@@ -1,11 +1,10 @@
 """Host API required Work Files tool"""
 import os
 import nuke
-from avalon import api
 
 
 def file_extensions():
-    return api.HOST_WORKFILE_EXTENSIONS["nuke"]
+    return [".nk"]
```

Comment:
should use `return api.HOST_WORKFILE_EXTENSIONS["nuke"]`

Reply:
agreed